arkane/statmech.py

```diff
@@ -688,7 +696,7 @@ def create_hindered_rotor_figure(self, angle, v_list, cosineRotor, fourierRotor,
         """
         try:
             import pylab
```

Comment:
Should we add `pylab` to our list of dependencies and conda environment files? If so we could move this to the general imports section and get rid of this code.

Reply:
Good point. Why are we using pyab and not matplotlib, anyway?